### DISCUSSION TITLE
Solidity contract rebase token

### DISCUSSION THREAD

**Discussion by JT202064** [PaulRBerg/hardhat-template#55] [Q&A] [answered]
Looking for a sample rebase contract possibly with a picture diagram on how the contract moves. Trying to wrap my head around insurance tokens

**Comment by PaulRBerg** [comment, marked as answer]
Hey @JT202064! I don't know of any such contract. You may have better luck asking for help in the [/r/ethdev](https://reddit.com/r/ethdev) subreddit.

For picture diagrams, you can use the [Solidity Visual Auditor](https://marketplace.visualstudio.com/items?itemName=tintinweb.solidity-visual-auditor) extension for VSCode.
